inc/styles.php

```diff
@@ -526,7 +526,8 @@ static function filter_css_object( $css, $panels_data, $post_id, $layout ) {
 			}
 
 			// Add in flexbox alignment to the main row element
-			if ( ! empty( $row['style']['cell_alignment'] ) ) {
+			$flexbox_align_styles = array('auto','flex-start', 'center', 'flex-end', 'stretch');
+			if ( ! empty( $row['style']['cell_alignment'] ) && in_array($row['style']['cell_alignment'], $flexbox_align_styles ) ) {
```

Comment:
The vertical alignment options are limited to flexbox because that's the layout system Page Builder is using. If you want to mix options related to flexbox layouts and non-flexbox layouts, you'd need to detect that the custom option isn't related to flexbox and therefore shouldn't be output as CSS `align-items` property. As I suggested previously, a custom field might help you do this. Another option is to use the `siteorigin_panels_css_object` filter and remove `cell_alignment` if it is set to one of your custom options. Then it won't output the CSS `align-items` property at all.